runtime/rococo/Cargo.toml

```diff
@@ -64,7 +64,7 @@ rococo-runtime-constants = { package = "rococo-runtime-constants", path = "./con
 runtime-common = { package = "polkadot-runtime-common", path = "../common", default-features = false }
 primitives = { package = "polkadot-primitives", path = "../../primitives", default-features = false }
 polkadot-parachain = { path = "../../parachain", default-features = false }
-runtime-parachains = { package = "polkadot-runtime-parachains", path = "../parachains", default-features = false }
+runtime-parachains = { package = "polkadot-runtime-parachains", path = "../parachains", default-features = false, features = ["vstaging"] }
```

Comment:
Enable runtime staging API in Rococo. It's still behind a feature flag which is disabled by default.

Reply:
Just for versi, but as far as I can see - they share single implementation. I mean there is no `versi-parachains` in the code so I've enabled it in Rococo.

This doesn't enable the client code though, so executing staging logic on Rococo shouldn't be possible. If this is a concern - I can make the above conditional too.